python/ray/serve/schema.py

```diff
@@ -437,6 +448,22 @@ def num_replicas_and_autoscaling_config(cls, values):
 
         return values
 
+    @root_validator
+    def validate_max_queued_requests(cls, values):
+        max_queued_requests = values.get("max_queued_requests", None)
+        if max_queued_requests is None or max_queued_requests == DEFAULT.VALUE:
+            return values
+
+        if not isinstance(max_queued_requests, int):
```

Comment:
I think pydantic will cast `max_queued_requests` to int automatically and if the value is not able to cast to int it should throw related error without needing to check again here.

Reply:
that's true i was maybe being too defensive :D let me add test cases for this one and remove this line